Apps/Playground/iOS/LibNativeBridge.mm

```diff
@@ -39,7 +40,11 @@ - (void)init:(void*)view width:(int)inWidth height:(int)inHeight xrView:(void*)x
     float height = inHeight;
     void* windowPtr = view;
     
-    graphics = Babylon::Graphics::CreateGraphics(windowPtr, static_cast<size_t>(width), static_cast<size_t>(height));
+    GraphicsConfiguration graphicsConfig = GraphicsConfiguration();
+    graphicsConfig.windowPtr = (__bridge WindowType)windowPtr;
```

Comment:
Can't the `(void*)view` parameter now be changed to `(MTKView*)view`?

Reply:
Yeah, it probably should be.

Reply:
changed.